paperoni/sources/scrapers/openreview.py

```diff
@@ -0,0 +1,228 @@
+from datetime import datetime
+from coleo import Option, tooled
+import openreview
+
+from ..model import (
+    Author,
+    DatePrecision,
+    Link,
+    Paper,
+    Release,
+    Topic,
+    Venue,
+    VenueType
+)
+from ...utils import QueryError
+
+class OpenReviewScraper:
+    @tooled
+    def query(
+        self,
+        # Author to query
+        # [alias: -a]
+        # [nargs: +]
+        author: Option = [],
+        # Title of the paper
+        # [alias: -t]
+        # [nargs: +]
+        title: Option = [],
+        # Maximal number of results per query
+        block_size: Option & int = 100,
+        # Maximal number of results to return
+        limit: Option & int = 10000,
+    ):
+        title = " ".join(title)
+
+        if author and title:
+            raise QueryError("Cannot query both author and title")
```

Comment:
That was a limitation for Semantic Scholar, but is it a limitation for OpenReview?

Reply:
Probably not but will check